clang/tools/clang-linker-wrapper/ClangLinkerWrapper.cpp

```diff
@@ -524,7 +1041,7 @@ std::unique_ptr<lto::LTO> createLTO(
       lto::createInProcessThinBackend(llvm::heavyweight_hardware_concurrency());
 
   Conf.CPU = Arch.str();
-  Conf.Options = codegen::InitTargetOptionsFromCodeGenFlags(Triple);
+  // Conf.Options = codegen::InitTargetOptionsFromCodeGenFlags(Triple);
```

Comment:
These are hacks I did for enabling thinLTO. I think they are not needed for the current workflow.

Reply:
Removed in latest commit. Thanks